src/box/lua/index.c

```diff
@@ -242,18 +242,30 @@ box_index_init_iterator_types(struct lua_State *L, int idx)
 static int
 lbox_index_iterator(lua_State *L)
 {
-	if (lua_gettop(L) != 4 || !lua_isnumber(L, 1) || !lua_isnumber(L, 2) ||
-	    !lua_isnumber(L, 3))
-		return luaL_error(L, "usage index.iterator(space_id, index_id, type, key)");
+	if (lua_gettop(L) != 5 || !lua_isnumber(L, 1) || !lua_isnumber(L, 2) ||
+	    !lua_isnumber(L, 3) || !lua_isstring(L, 5))
+		return luaL_error(L, "usage index.iterator(space_id, index_id, "
+				     "type, key, after)");
+
 
 	uint32_t space_id = lua_tonumber(L, 1);
 	uint32_t index_id = lua_tonumber(L, 2);
 	uint32_t iterator = lua_tonumber(L, 3);
 	size_t mpkey_len;
 	const char *mpkey = lua_tolstring(L, 4, &mpkey_len); /* Key encoded by Lua */
 	/* const char *key = lbox_encode_tuple_on_gc(L, 4, key_len); */
-	struct iterator *it = box_index_iterator(space_id, index_id, iterator,
-						 mpkey, mpkey + mpkey_len);
+	size_t packed_pos_size;
+	const char *packed_pos = lua_tolstring(L, 5, &packed_pos_size);
```

Comment:
Why don't you use `lbox_normalize_position` here, like `lbox_select` does?

Reply:
I did it because I wanted to save unpacked position in Lua. It is not actual anymore, so I rewrote it.